### PR TITLE
Remove to force including plugins sources into tests

### PR DESCRIPTION
**Pull request by oscargm98**

### CODE REVIEW

File: plugins/codecs/ndlz/CMakeLists.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ set(SOURCES ${SOURCES} ndlz.c ndlz.h ndlz-private.h ndlz4x4.c ndlz4x4.h ndlz8x8.
```
Comment (FrancescAlted) [inline plugins/codecs/ndlz/CMakeLists.txt:3]: You don't need `set(SOURCES...)` line anymore, right?

**Review by FrancescAlted — CHANGES_REQUESTED** [review]

File: plugins/codecs/ndlz/CMakeLists.txt (hunk 1 above)
Comment (FrancescAlted) [reply plugins/codecs/ndlz/CMakeLists.txt:3]: And the same goes for other cmake files.

Comment (oscargm98) [reply plugins/codecs/ndlz/CMakeLists.txt:3]: Then no plugin source will be distributed. If we want plugins to be in the distribution library, then they must be into SOURCES first.

Comment (FrancescAlted) [reply plugins/codecs/ndlz/CMakeLists.txt:3]: Oh, my mistake.  So if I understand correctly, the updated `$SOURCES` variable is used *outside* of this cmake file, right?  If so, I have no objections.

Comment (oscargm98) [reply plugins/codecs/ndlz/CMakeLists.txt:3]: That is what I think. This evening I make sure of it and I confirm you.

**Pull request closed without merging**